internal/sdks/sdk_instructions/go.md

```diff
@@ -99,9 +99,10 @@ func main() {
 
 Now that your application is ready, run the application to see what value we get.
 ```shell
+export LAUNCHDARKLY_SDK_KEY=1234567890abcdef
```

Comment:
the file gets the environment variable but we didn't actually set it anywhere. this could probably go at the top or here, doesn't matter much where